tests/testthat/test-ops.R

```diff
@@ -0,0 +1,23 @@
+test_that("can access operators via ops object", {
+  # matmul, default use
+  res <- ops$aten$matmul(torch::torch_ones(5, 4), torch::torch_rand(4, 5))
+  expect_equal(length(res), 1)
```

Comment:
Not necessarily in this PR, but I wonder if given the return type of the schema that was used, if it would be possible to 'unlist' the output. For instance, we know that `ops$aten$matmul` returns a `TensorType` (from the schema). Could we in this case instead of returning a list, return the Tensor directly?

Reply:
Oh yes, that makes a lot of sense! Will do in this one ...